src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleListEntryController.cs

```diff
@@ -5,14 +5,14 @@
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using VirtoCommerce.CatalogModule.BulkActions.Models;
```

Comment:
Need to refactor the same way as it was done in v2 to avoid code duplication:
https://github.com/VirtoCommerce/vc-module-catalog/commit/9e714d341114d999f3c4a6c580950894e22ac78e